tests/validation/tests/v3_api/test_cluster_templates.py

```diff
@@ -70,6 +97,26 @@ def get_cluster_config(k8sversion):
             "enabled": "true",
             "type": "localClusterAuthEndpoint"
         },
+        "scheduledClusterScan": {
+            "enabled": "true",
+            "scanConfig": {
+                "cisScanConfig": {
+                    "debugMaster": "false",
+                    "debugWorker": "false",
+                    "overrideBenchmarkVersion": "rke-cis-1.4",
```

Comment:
Should we make use of `from .common import CIS_SCAN_PROFILE` here ?

Reply:
Done